qfieldcloud_sdk/cli.py

```diff
@@ -736,3 +736,143 @@ def members_patch(
         log(
             f'Member "{membership["member"]}" changed role in organization "{membership["organization"]}" to role "{membership["role"]}".'
         )
+
+
+@cli.command(name="teams-list", short_help="Get a list of organization teams.")
```

Comment:
```suggestion
@cli.command(short_help="Get a list of organization teams.")
```
I think there is no need for explicit `name` kwarg?

Reply:
That's right!